src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -97,6 +97,9 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.10" />
+    <SDKReference Include="WindowsMobile, Version=10.0.19041.0">
```

Comment:
Do we need this?

Reply:
Yes. Without it, our project wouldn't compile. I think, it's needed to change status bar color on mobile devices in UWP applications.

Reply:
But there's no Windows Mobile (RIP) we need to review it and make sure that we aren't missing anything

Reply:
Should I remove code related to Windows Mobile?

Reply:
Here is type `WindowsMobile` SDK is needed for: https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.statusbar

Reply:
@pictos Should I remove code related to Windows Phone / Windows Mobile?

Reply:
Discussed offline. Dropped support for Windows Mobile